### PR TITLE
fix appsec issue related to iat and product services

### PR DESCRIPTION
**Pull request by 065vaibhav**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--{repo}--{owner}.hlx.live/
- After: https://<branch>--{repo}--{owner}.hlx.live/

### CODE REVIEW

File: common/makeRestAPI.js [hunk 1]
```diff
@@ -34,18 +34,28 @@ function hideLoaderGif() {
...
+        'Content-type': 'text/plain',
+        Accept: 'application/json',
+      };
+
+      headers.iat = window ? btoa(currentDate.getTime()) : '';
       return fetch(url, {
         method,
         body: payload ? JSON.stringify(payload) : null,
         mode: 'cors',
         headers: {
```
Comment (roym1234) [inline common/makeRestAPI.js:52]: Remove this header

File: common/makeRestAPI.js [hunk 2]
```diff
@@ -34,18 +34,28 @@ function hideLoaderGif() {
...
           'Content-type': 'text/plain',
           Accept: 'application/json',
+          iat: window ? btoa(currentDate.getTime()) : '',
+
         },
+        credentials: 'same-origin',
```
Comment (roym1234) [inline common/makeRestAPI.js:58]: Remove the credentials

File: common/makeRestAPI.js [hunk 3]
```diff
@@ -60,6 +70,7 @@ async function fetchJsonResponse(url, payload, method, loader = false) {
       headers: {
         'Content-type': 'text/plain',
         Accept: 'text/plain',
+        credentials: 'include',
```
Comment (roym1234) [inline common/makeRestAPI.js:73]: Remove the credentials

File: common/makeRestAPI.js (hunk 1 above)
Comment (065vaibhav) [reply common/makeRestAPI.js:52]: fix

File: common/makeRestAPI.js (hunk 3 above)
Comment (065vaibhav) [reply common/makeRestAPI.js]: fix

**Pull request merged**